### DISCUSSION TITLE
send document file

### DISCUSSION THREAD

**Discussion by rezakarbasi** [python-telegram-bot/python-telegram-bot#2297] [Q&A] [answered]
Hi,
how can i send a '.pdf' file via this library?

**Comment by RememberTheAir** [comment, marked as answer]
The wiki has a quite detailed chapter about working with files and medias: [Working with files and media](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Code-snippets#working-with-files-and-media)

In short, to send a file from disk, you have to open it and pass the bytes object to the [`Bot.send_document`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.bot.html#telegram.Bot.send_document) method:

```py
with open("some_file.txt", "rb") as f:
  bot.send_document(chat_id, f)
```

**Comment by MORTEZA-KING-007** [comment]
Hello how can i send i file like jpg mp4 via python-telegram-bot

**Comment by axait** [comment]
unsubscribe
